### PR TITLE
Uptake new rancher 2.6.3 bfs images

### PR DESCRIPTION
**Pull request by markxnelson**
Signed-off-by: Mark Nelson <mark.x.nelson@oracle.com>

# Description

This PR uptakes new Rancher 2.6.2 bfs images, and is the first part of VZ-5241.  This PR introduces the following changes:

- Rancher Server is upgraded to 2.6.3
- Rancher Server and Agent are built from source by Oracle
- Container images use Oracle Linux as the base image
- Some security fixes are implemented
- The Rancher Helm chart is updated to the upstream 2.6.3 charts and variables are replaced with realized values
- Verrazzano install and install tests are updated to reflect changes in Rancher 2.6.3, in particular the fleet agent is moved from the cattle-fleet-system namespace to the cattle-fleet-local-system namespace, the rancher-operator-system namespace is removed and the rancher operator is no longer deployed

This PR does not address updating/migrating Rancher when upgrading from earlier versions of Verrazzano to 1.3/master.  The remaining container images are not updated to be built from source/used yet.  Additional work is required to customize the first login flow (skip the bootstrap password page and set the server URL).

Contributes to VZ-2451

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/controllers/verrazzano/component/rancher/rancher.go [hunk 1]
```diff
@@ -27,7 +28,7 @@ const (
 	fleetControllerDeployment = "fleet-controller"
 	gitjobDeployment          = "gitjob"
 	rancherWebhookDeployment  = "rancher-webhook"
-	rancherOperatorDeployment = "rancher-operator"
+	//rancherOperatorDeployment = "rancher-operator"
```
Comment (kminder) [inline platform-operator/controllers/verrazzano/component/rancher/rancher.go:31]: Can you add a comment for why this is being removed?

File: platform-operator/controllers/verrazzano/component/rancher/rancher.go [hunk 2]
```diff
@@ -17,8 +17,9 @@ import (
 
 // Constants for Kubernetes resource names
 const (
-	fleetSystemNamespace      = "fleet-system"
-	OperatorNamespace         = "rancher-operator-system"
+	fleetSystemNamespace      = "cattle-fleet-system"
```
Comment (kminder) [inline platform-operator/controllers/verrazzano/component/rancher/rancher.go:20]: What are the upgrade implications of this?
Can you comment why it is being changed?

File: platform-operator/controllers/verrazzano/component/rancher/rancher_component.go [hunk 3]
```diff
@@ -174,9 +174,9 @@ func (r rancherComponent) PreInstall(ctx spi.ComponentContext) error {
 	vz := ctx.EffectiveCR()
 	c := ctx.Client()
 	log := ctx.Log()
-	if err := createRancherOperatorNamespace(log, c); err != nil {
-		return err
-	}
+	//if err := createRancherOperatorNamespace(log, c); err != nil {
```
Comment (pfmackin) [inline platform-operator/controllers/verrazzano/component/rancher/rancher_component.go:177]: looks like we are getting rid of the rancher operator.  If so we should delete the code.

File: platform-operator/controllers/verrazzano/component/rancher/rancher.go (hunk 2 above)
Comment (markxnelson) [reply platform-operator/controllers/verrazzano/component/rancher/rancher.go:20]: that's an excellent question - that's why i asked @pfmackin  to review this :)

Comment (markxnelson) [reply platform-operator/controllers/verrazzano/component/rancher/rancher.go:23]: added a comment to explain - upgrade will be handled in another PR

**Review by pfmackin — APPROVED** [review]

**Pull request merged**